user-guide/running-tests.md

```diff
@@ -129,37 +129,39 @@ Executing `bench` the `--help` or `-h` switches will print descriptions of the s
 
 ``` bash
 > bench --help
-Usage: bench [options] [paths]
+Usage: ./script/bench [options] [paths]
```

Comment:
Doesn't this depend on a particular specialization, ie: putting the executable (or symlink) in the "script" directory? Is this something that's going to apply to the greatest number of users?

Reply:
Thanks! Corrected.